components/Player/Players.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Player from "./Player";
+import { useSelector } from "react-redux";
+import type { RootState } from "../../reducers/store";
+
+
+export default function Players() {
+    const gameState = useSelector((state: RootState) => {
+        return state.game
+    });
+
+    return (
+        <>
+            {displayPlayers(gameState.numPlayers)}
+        </>
+    )
+}
+
+function displayPlayers(numPlayers: number) {
+    let players: JSX.Element[] = []
+    for(let i = 0; i < numPlayers; i++) {
+      players.push(
+        <Player key={i} color={'white'} number={i}/>
+      )
+    }
+    return players;
+}
```

Comment:
Add a new line at the bottom

Reply:
Also I should find a better way to do this other than just a for loop.  Maybe
```
return Array(numPlayers).keys().map(key => {
   return <Player key={key} color={color} number={key}/>
```
Also need to find a way to get the players color.

Reply:
Changed to a map.